docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -178,6 +184,20 @@ filter expression in order to satisfy the policy. The `count` field of a require
 policy is a positive integer value that specifies the minimum amount of users
 this policy requires.
 
+### On leave
+
+The `on_leave` string option in require policies is used to define what happens when a user leaves a session, causing a policy to no longer be satisfied.
+
+There are two possible actions to take in this scenario:
+1. Terminate the session and disconnect all participants, corresponding to the `"terminate"` value.
+2. Pause the session and stop any input/output streaming until the policy is satisfied again, corresponding to the `"pause"` value.
+
+By default, Teleport treats an empty string in this field as the same as `terminate`.
+That is, the session is terminated instantly and all participants are disconnected.
+
+If all require policies attached to the session owner are set to `"pause"`, the session will instead pause
+but the session will remain open. This discards all input from session participants and buffers output until the session can resume.
+
```

Comment:
Does it buffer or discard output?

Buffering output while a moderator is not present is a security risk, as the user can enter a command that will immediately execute when the moderator returns.

Reply:
Did you think of input? I don't see how buffering output (that is, stdout from the shell to the user) could pose a security risk?

Reply:
I believe the wording does make it clear that all input is indeed discarded.